lib/std/Target/spirv.zig

```diff
@@ -1,297 +1,26 @@
-    ImageFootprintNV,
-    FragmentBarycentricNV,
-    ComputeDerivativeGroupQuadsNV,
-    FragmentDensityEXT,
-    ShadingRateNV,
-    GroupNonUniformPartitionedNV,
-    ShaderNonUniform,
-    ShaderNonUniformEXT,
-    RuntimeDescriptorArray,
-    RuntimeDescriptorArrayEXT,
-    InputAttachmentArrayDynamicIndexing,
-    InputAttachmentArrayDynamicIndexingEXT,
-    UniformTexelBufferArrayDynamicIndexing,
-    UniformTexelBufferArrayDynamicIndexingEXT,
-    StorageTexelBufferArrayDynamicIndexing,
-    StorageTexelBufferArrayDynamicIndexingEXT,
-    UniformBufferArrayNonUniformIndexing,
-    UniformBufferArrayNonUniformIndexingEXT,
-    SampledImageArrayNonUniformIndexing,
-    SampledImageArrayNonUniformIndexingEXT,
-    StorageBufferArrayNonUniformIndexing,
-    StorageBufferArrayNonUniformIndexingEXT,
-    StorageImageArrayNonUniformIndexing,
-    StorageImageArrayNonUniformIndexingEXT,
-    InputAttachmentArrayNonUniformIndexing,
-    InputAttachmentArrayNonUniformIndexingEXT,
-    UniformTexelBufferArrayNonUniformIndexing,
-    UniformTexelBufferArrayNonUniformIndexingEXT,
-    StorageTexelBufferArrayNonUniformIndexing,
-    StorageTexelBufferArrayNonUniformIndexingEXT,
-    RayTracingNV,
-    VulkanMemoryModel,
-    VulkanMemoryModelKHR,
-    VulkanMemoryModelDeviceScope,
-    VulkanMemoryModelDeviceScopeKHR,
-    PhysicalStorageBufferAddresses,
-    PhysicalStorageBufferAddressesEXT,
-    ComputeDerivativeGroupLinearNV,
-    RayTracingProvisionalKHR,
-    CooperativeMatrixNV,
-    FragmentShaderSampleInterlockEXT,
-    FragmentShaderShadingRateInterlockEXT,
-    ShaderSMBuiltinsNV,
-    FragmentShaderPixelInterlockEXT,
-    DemoteToHelperInvocationEXT,
-    SubgroupShuffleINTEL,
-    SubgroupBufferBlockIOINTEL,
-    SubgroupImageBlockIOINTEL,
-    SubgroupImageMediaBlockIOINTEL,
-    RoundToInfinityINTEL,
-    FloatingPointModeINTEL,
-    IntegerFunctions2INTEL,
-    FunctionPointersINTEL,
-    IndirectReferencesINTEL,
-    AsmINTEL,
-    AtomicFloat32MinMaxEXT,
-    AtomicFloat64MinMaxEXT,
-    AtomicFloat16MinMaxEXT,
-    VectorComputeINTEL,
-    VectorAnyINTEL,
-    ExpectAssumeKHR,
-    SubgroupAvcMotionEstimationINTEL,
-    SubgroupAvcMotionEstimationIntraINTEL,
-    SubgroupAvcMotionEstimationChromaINTEL,
-    VariableLengthArrayINTEL,
-    FunctionFloatControlINTEL,
-    FPGAMemoryAttributesINTEL,
-    FPFastMathModeINTEL,
-    ArbitraryPrecisionIntegersINTEL,
-    UnstructuredLoopControlsINTEL,
-    FPGALoopControlsINTEL,
-    KernelAttributesINTEL,
-    FPGAKernelAttributesINTEL,
-    FPGAMemoryAccessesINTEL,
-    FPGAClusterAttributesINTEL,
-    LoopFuseINTEL,
-    FPGABufferLocationINTEL,
-    USMStorageClassesINTEL,
-    IOPipesINTEL,
-    BlockingPipesINTEL,
-    FPGARegINTEL,
-    AtomicFloat32AddEXT,
-    AtomicFloat64AddEXT,
-    LongConstantCompositeINTEL,
+    v1_6,
+    int8,
+    int16,
+    int64,
+    float16,
+    float64,
+    addresses,
+    matrix,
+    kernel,
+    generic_pointer,
+    vector16,
+    shader,
```

Comment:
Are these the same features as LLVM?

Reply:
LLVM doesn't define any features (yet?).
https://github.com/llvm/llvm-project/blob/5c93eb56dc9bc0c0210483cdd5d31e6b6580454f/llvm/lib/Target/SPIRV/SPIRV.td

Reply:
Ah, that's odd. I recall that there were at least some features in the past, and also I'm sure that the backend can deal with extensions... I've seen it being used in some way.